bcda/cclf/cclf.go

```diff
@@ -559,6 +561,30 @@ func orderACOs(cclfMap *map[string]map[int][]*cclfFileMetadata) []string {
 	return acoOrder
 }
 
+func getPriorityACOs(db *gorm.DB) []string {
+	const query = `
+	SELECT trim(both '["]' from g.x_data::json->>'cms_ids') "aco_id" 
+	FROM systems s JOIN groups g ON s.group_id=g.group_id 
+	WHERE s.deleted_at IS NULL AND g.group_id IN (SELECT group_id FROM groups WHERE x_data LIKE '%A%' and x_data NOT LIKE '%A999%') AND
+	s.id IN (SELECT system_id FROM secrets WHERE deleted_at IS NULL);
+	`
+	
+	defaultPriority := strings.Split(os.Getenv("CCLF_PRIORITY_ACO_CMS_IDS"), ",")
```

Comment:
Is this even necessary anymore if we are doing the check in real time?  Might be too many knobs having both capabilities.  I'd say we default to random order if the query fails (and log it so its in splunk).

Reply:
Good idea, I was thinking that a fallback would be nice, but agreed that we may have too many options at our controls. Will remove.

Since we're pulling out this value, I'll go ahead and put up a PR for the ops change as well.